scripts/helper/functions.sh

```diff
@@ -59,6 +59,38 @@ preflight_checks()
 
 }
 
+postflight_checks()
+{
+  # Verify no containers have Exited
+  if [[ $(docker-compose ps | grep Exit) ]]; then
+    echo -e "\nWARNING: at least one Docker container unexpectedly exited. Please troubleshoot, see https://docs.confluent.io/current/tutorials/cp-demo/docs/index.html#troubleshooting"
+  fi
+
+  # Validate connectors are running
+  # The Replicator connector may come fully UP after start script ends, so ignore that connector here (to not add time to start script)
+  if [[ "$VIZ" == "true" ]]; then
+    connectorList="wikipedia-sse elasticsearch-ksqldb"
+  else
+    connectorList="wikipedia-sse"
+  fi
```

Comment:
Could this be made a little more future proof by querying for the connector list from the connect worker and then querying for each status based on that result?

Reply:
What if the connector was never successfully submitted, the connect worker wouldn't know about it.  The thinking is that the validation script should be directive, not let it to chance that the connector(s) were submitted properly.

Reply:
I thought of this as well, but if the connector is missing in this case, are you going to get a 404 from the endpoint and will the STATE != "RUNNING" show something meaningful?  

Additionally, i'm thinking maybe it's too late for this error, and that situation should be highlighted by the submit script?

Reply:
We don't have to hold up this PR for this feedback, but we might want to double check what happens if the connector is not there (test w/ an invalid name maybe?).

Reply:
Refactored code to check status of connectors configured in connect